### DISCUSSION TITLE
Order of tabs listed underneath Window menu seems wrong

### DISCUSSION THREAD

**Discussion by curiousier-george** [ghostty-org/ghostty#3505] [Feature Requests, Ideas] [answered]
Thanks for making Ghostty!

After changing the order of the tabs via dragging with the mouse, the order of the tabs listed at the bottom of the Mac Window menu doesn't reflect the new order of the tabs.

**Comment by mitchellh** [comment]
Can you share your config? A screenshot would help too. I don't see this on my end and need to figure out why.

**Reply by carlcotner** [reply]
Sure. Here's my config:

```
background = 000000
shell-integration = none
shell-integration-features = no-cursor
cursor-style = underline
```

and a screenshot:
![ghostty](https://github.com/user-attachments/assets/7ce96252-5298-48c6-9e42-2cdb0b7c9e6a)

**Reply by mitchellh** [reply, marked as answer]
Oh that order. That's maintained by macOS, we don't manage that menu at all. I can poke at it but that's very odd.

**Comment by carlcotner** [comment]
Hmm, I was wondering whether it was related to some of the other reports of window/tab issues because of macOS bugs.
